README.md

```diff
@@ -1084,6 +1085,27 @@ module.exports = {
 };
 ```
 
+## `compareThreshold`
+
+By default, a shallow comparison when `happo compare` is called. If two images
+are different on one pixels or more, it will be reported as a diff. If you set
+a `compareThreshold`, a deep comparison will be performed instead, where
+individual pixels are diffed to compute a total euclidean distance between the
+two images. The diff value will be between 0 and 1. A value close to 1 means
+most pixels are different. A value close to 0 means images are very similar.
+
+```js
+module.exports = {
+  compareThreshold: 0.005,
+};
+```
+
+Finding the right threshold isn't always trivial. One thing to help in the
+process is to make a few dry-run comparison. Find one or a few comparisons (via
+https://happo.io/dashboard) and run `happo compare <sha1> <sha2> --dry-run` on
+the shas and look at what's being logged to figure out what threshold value you
+want to use.
```

Comment:
I think another change that would help people get a feel for what different values mean would be to expose the euclidean distance on the report page when looking at a diff.

Reply:
This is a good idea. I'll add a todo to change this in the happo backend.